docs/Configuration.md

```diff
@@ -627,30 +627,30 @@ For more details on break build, please refer to [Thresholds and policies](https
+| `isBranchedIncremental`                | false                 | No                                  | Yes     | Yes               | If this option is true Cx-flow will do incremental scan for first time created branched project.                                                                                                                                                                                                                                                                                                                                                                                        |
+| `cancelInpregressScan`                 | false                 | No                                  | Yes     | Yes               | If a scan timeout occurs and the user requests to cancel the in-progress scan, set this boolean variable to true.                                                                                                                                                                                                                                                                                                                                                                       |
+| `token `                               |                       | No                                  | Yes     | Yes               | The user can pass a token instead of using a username and password. Note: The default token is valid for 24 hours. Either provide a new token before the 24-hour period expires or update the token's lifetime from the SAST backend table.                                                                                                                                                                                                                                             |
+| `enableTokenLogin `                    | false                 | No                                  | Yes     | Yes               | After enabling this parameter, CXFlow will communicate with the SAST API using a token instead of a username and password. It is mandatory to pass the token once this parameter is enabled.                                                                                                                                                                                                                                                                                            |
 
```

Comment:
We might need to update Documentation for pre-requisite page where Java version, SAST version, Jira, Gitlab, Github etc. are specified?